### PR TITLE
Support @ember/string v4

### PR DESCRIPTION
**Pull request by Techn1x**
`@ember/string` 4 was released recently, so we need to widen this to allow applications to use it

### CODE REVIEW

File: pnpm-lock.yaml
Comment (Techn1x) [inline pnpm-lock.yaml:1]: this large diff is due to the lockfile being last generated under pnpm v6, but I have run pnpm install via pnpm v9

**Review by jelhan — APPROVED** [review]
Thanks a lot!

**Comment by Techn1x** [comment]
thanks for merging! Will keep an eye out for next release 🤓

**Comment by jelhan** [comment]
@Techn1x I have released it as [v4.4.0](https://github.com/jelhan/ember-style-modifier/releases/tag/v4.4.0) just now. Please excuse the delay.

**Pull request merged**